packages/shared/providers.yaml

```diff
@@ -534,7 +533,6 @@ close:
         - offline_access
     proxy:
         base_url: https://api.close.com/api
-        docs: https://developer.close.com/
```

Comment:
why removing this?

Reply:
it's a leftover when we changed to point to our doc